code/__DEFINES/dcs/signals.dm

```diff
@@ -434,6 +434,8 @@
 #define COMSIG_OBJ_DEFAULT_UNFASTEN_WRENCH "obj_default_unfasten_wrench"
 ///from base of /turf/proc/levelupdate(). (intact) true to hide and false to unhide
 #define COMSIG_OBJ_HIDE	"obj_hide"
+/// from /obj/item/toy/crayon/spraycan/afterattack: (color_is_dark)
+#define COMSIG_OBJ_PAINTED "obj_painted"
```

Comment:
Why send only if it's dark? wouldn't it add more support to send the color array itself? HSL, RGB, ect.

Reply:
I can do that. The idea behind it was so that `DARK_COLOR_LIGHTNESS_THRESHOLD` is defined only where the spray cans are. Sending out the color array would require moving that define somewhere more up.

Reply:
I guess it depends on if we want that support or not. Up to you, I'll take a look at this in the morning and if you're happy with it merge it.

Reply:
I think for now this is fine enough.